stripper/ze_aurora_tower_v6_6.cfg

```diff
@@ -1,3 +1,105 @@
+;filter unpacked particle
+filter:
+{
+	"classname" "info_particle_system"
+	"targetname" "PrimeMecha_Model_parti"
+}
+
+filter:
+{
+	"classname" "logic_timer"
+	"targetname" "PrimeMecha_Model_minitimer"
+}
+
+modify:
+{
+	match:
+	{
+		"classname" "game_ui"
+		"targetname" "PrimeMecha_UI_mini"
+	}
+	delete:
+	{
+		"PressedAttack2" "PrimeMecha_Model_minitimerEnable1.7-1"
+		"PressedAttack2" "PrimeMecha_Model_minitimerDisable11.7-1"
+	}
+}
+
+;fix frozen zombies getting boosted after being unfrozen
+modify:
+{
+	match:
+	{
+		"classname" "trigger_multiple"
+		"targetname" "HumanItem_HealBlade_Skill2_Relics_FreezeTrigger"
+	}
+	insert:
+	{
+		"OnStartTouch" "!activatorRunScriptCodeself.SetVelocity(Vector(0,0,0))5.8-1"
+	}
+}
+
+;patch tp avoidance spot
+modify:
+{
+	match:
+	{
+		"classname" "trigger_teleport"
+		"targetname" "FallOutMap_Zombie_TeleArea"
+	}
+	delete:
+	{
+		"model" "*27"
+	}
+}
+
+modify:
+{
+	match:
+	{
+		"classname" "trigger_hurt"
+		"targetname" "FallOutMap_CT_KillArea"
+	}
+	delete:
+	{
+		"model" "*28"
+	}
+}
+
+modify:
+{
+	match:
+	{
+		"classname" "trigger_teleport"
+		"targetname" "Lv3FinalRun_NoFreeFall_Teleport"
+		"origin" "-7020.14 -170.5 16"
+	}
+	delete:
+	{
+		"model" "*497"
+	}
+}
```

Comment:
You're gonna need to give this trigger a temporary custom targetname, you are inputting to 5 different `Lv3FinalRun_NoFreeFall_Teleport` triggers below.